core/server/worker/src/main/java/alluxio/worker/block/BlockMasterClient.java

```diff
@@ -139,6 +139,10 @@ public long getId(final WorkerNetAddress address) throws IOException {
     }, LOG, "GetId", "address=%s", address);
   }
 
+  /**
+   * Converts the block list map to proto.
+   * The output list will not contain duplicate keys.
+   * */
```

Comment:
I think this comment line looks off?

Reply:
Well yeah it looks a bit strange where a "list" has "keys". I'm just trying to say that the map is flattened into a list so when the list is converted to a map on the other side, there will be no key collision, and this is the guaranteed behavior.

Reply:
Well yeah it looks a bit strange where a "list" has "keys". I'm just trying to say that the map is flattened into a list so when the list is converted to a map on the other side, there will be no key collision, and this is the guaranteed behavior.

Reply:
@gpang I tried to rephrase, does it make more sense now?

Reply:
This looks OK now.